### PR TITLE
Updated german translation

### PR DESCRIPTION
**Pull request by uDEV2019**
- fixed some mistranslations
- entered missing conjunctions
- added logical translations, e.g. openNewMail

### CODE REVIEW

**Comment by oliverschwendener** [comment]
Thanks for your contribution! Translations recently just changed. Can you merge the current dev back to your branch and update the translations? Thanks!

**Comment by uDEV2019** [comment]
Updated it

File: src/common/translation/german-translation-set.ts [hunk 1]
```diff
@@ -109,24 +109,28 @@ export const germanTranslationSet: TranslationSet = {
...
+    appearanceSettingsResetWarningMessage: "Du bist dabei alle Erscheinungsbildeinstellungen auf Standard zurückzusetzen. Willst Du fortfahren?",
+    appearanceSettingsAllowTransparentBackground: "Transparenten Hintergrund erlauben",
     appearanceSettingsFontFamily: "Schriftart",
     appearanceSettingsUserInputBorderRadius: "Benutzereingabe Randradius",
     appearanceSettingsUserInputBottomMargin: "Benutzereingabe unterer Rand (in Pixel)",
     appearanceSettingsSearchResultsBorderRadius: "Suchergebnisse Rahmenradius",
     appearanceSettingsScrollbarBorderRadius: "Randradius der Bildlaufleiste",
-    appearanceSettingsBorderRadiusDescription: "Dies ist ein CSS-Wert, daher müssen Sie 'px' wie diesen '10px' oder separate Ecken wie diesen '10px 0px 10px 5px' hinzufügen.",
-
+    appearanceSettingsBorderRadiusDescription: "Es handelt sich um einen CSS-Wert, daher musst Du ein 'px' wie bei '10px' oder getrennt wie '10px 0px 10px 5px' hinzufügen.",
```
Comment (oliverschwendener) [inline src/common/translation/german-translation-set.ts:119]: Diesen Satz könnte man noch verbessern, oder? 😄

**Comment by oliverschwendener** [comment]
This branch still has conflicts with the current `dev` branch. Can you resolve them, otherwise I can't merge. Thanks!

File: src/common/translation/german-translation-set.ts (hunk 1 above)
Comment (oliverschwendener) [reply src/common/translation/german-translation-set.ts:119]: Hab grad gesehen, dass die englische Übersetzung auch nicht grad perfekt ist. Lassen wir's so. 👍

**Comment by uDEV2019** [comment]
On conflict resolve it added the four new strings twice. I had to update again an removed the duplicates.

**Comment by uDEV2019** [comment]
No correct german translation needed.

**Pull request closed without merging**